livesync/indico_livesync/handler.py

```diff
@@ -88,12 +79,17 @@ def connect_signals(plugin):
     plugin.connect(signals.attachments.attachment_updated, _attachment_changed)
 
 
-def _moved(obj, old_parent, new_parent, **kwargs):
+def _moved_event(obj, old_category, **kwargs):
```

Comment:
Does it really matter though? It's a signal handler, not some API.
Anyway, It looks like the signal could be connected directly to `_moved`